CoEpi/ui/onboarding/OnboardingButton.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+class OnboardingButton: UIButton {
+    
+    override func awakeFromNib() {
+        
+        super.awakeFromNib()
+        
+        backgroundColor = Constants.color.coepiPrimaryButton
+        layer.cornerRadius = 5
+        contentEdgeInsets = UIEdgeInsets(top: 20,
+                                         left: 40,
+                                         bottom: 20,
+                                         right: 40)
+        
+        setTitleColor(.white, for: .normal)
```

Comment:
Can't all these attributes be set in the xib?
See https://stackoverflow.com/a/15839914/930450 for corner radius

Reply:
They can, however this button is going to get reused in future. So, it removed the need for manually updating the various nibs across the project. But I'm happy to remove it if preferred

Reply:
It's a fair point, but styling isn't a good reason for subclassing. We should create reusable styles. I'll add a TODO for now.

Reply:
It's a fair point, but subclassing isn't a good fit for styling, in particular because it composes poorly. We should use either reusable style objects/functions or protocol extensions. I'll add a TODO for now.